packages/nuxt/src/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "../tsconfig.json",
```

Comment:
should this extend from `../tsconfig.types.json` instead? No worries, if it works like this I'm also fine with it!

Reply:
It just needs the `declaration` set to `true`, so I didn't extend from there.